torch/distributed/_tensor/ops/_pointwise_ops.py

```diff
@@ -580,6 +580,7 @@ def linear_pointwise_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> Strategy
     aten._foreach_cos_.default,
     aten._foreach_log.default,
     aten._foreach_log_.default,
+    aten._amp_foreach_non_finite_check_and_unscale_.default,
```

Comment:
after switching to custom op handler, is this line still necessary? does it make a difference with/without this line?

Reply:
Removing the registration here could cause the error, fail to `unwrap_to_op_info`